python/ingestion/cdc_wonder_utils.py

```diff
@@ -139,6 +139,7 @@ def load_cdc_df_from_data_dir(
         if geo_level == NATIONAL_LEVEL:
             topic_df[state_code_col] = US_FIPS
 
+        topic_df[YEAR_COL] = topic_df[YEAR_COL].astype(int).astype(str)

```

Comment:
would it be better to just add YEAR_COL as a str to the DTYPE above so it gets read in as a string from the beginning?